build-libpostal.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -Eeuo pipefail
+
+cp -r "${SRC_DIR}"/libpostal/* .
+
+CFLAGS="-fPIC -lstdc++"
+CXXFLAGS="-fPIC"
+./bootstrap.sh
+
+ARCH_FLAGS=""
+case ${ARCH_TRIPLET} in
+"arm-linux-gnueabihf" )
+    ARCH_FLAGS="--with-cflags-scanner-extra=-marm"
+    ;;
+"aarch64-linux-gnu" )
+    # TODO what do we need to pass here for arm64?
+    ARCH_FLAGS="--with-cflags-scanner-extra=-marm"
```

Comment:
Any idea which flag is needed for arm64 builds? It does neither work with nor without `-marm`.

Reply:
Ok, somehow [managed](https://github.com/openvenues/libpostal/issues/349#issuecomment-575805153) to make it work.